packages/myst-to-typst/src/utils.ts

```diff
@@ -49,8 +49,8 @@ const textOnlyReplacements: Record<string, string> = {
   '©': '#emoji.copyright ',
   '®': '#emoji.reg ',
   '™': '#emoji.tm ',
-  '<': '\\< ',
-  '>': '\\> ',
+  '<': '\\<',
+  '>': '\\>',
```

Comment:
Making this change did not feel good - I'm not sure why we were previously adding a space after these characters. However, for the examples I was looking at, the addition of this space looked very wrong, and simply removing it here fixed the output.

@rowanc1 perhaps you have some recollection of this...?

Reply:
Lets go with an oversight/bug for now.